### DISCUSSION TITLE
what is the différence

### DISCUSSION THREAD

**Discussion by spinelbusiness** [JoomCoder-com/JoomCCK#249] [General]
Hello what are differences between joomcck free and joomcck extended ?

**Comment by CMS-CORP** [comment]
curious to know too, i need to purchase licence to test for my customers

**Comment by Giorgi625** [comment]
Free version has only few fields available, All fields are available in Extended version. 
 Fields listed here are only in Extended version and other addons listed on this page
https://www.joomcoder.com/joomla-extensions/9-components/24-joomcck

**Reply by CMS-CORP** [reply]
ok it's clearer now, thx

**Reply by CMS-CORP** [reply]
joomcck is very powerful, very complicated this kind of solution in wordpress especially with the system of front end + relationship parents / child, ect (by the admission of the team of toolset and formidable).

**Reply by Giorgi625** [reply]
Yes it is very powerful, before I decided to use it I tested all other CCK extensions which are present in Joomla and none of them can come close to JoomCCK.

**Comment by andiamine** [comment]
> Hello what are differences between joomcck free and joomcck extended ?

JoomCCK extended add some extra fields, modules and features to the free community version
